nixos/modules/misc/ids.nix

```diff
@@ -356,6 +356,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      jotta-cli = 327;
```

Comment:
Please explain why we cannot auto allocate a user id.

Reply:
The header comment of this file says:

> We only add static uids and gids for services where it is not feasible to change uids/gids on service start, for example a service with a lot of files.

I interpreted "a service with a lot of files" as "a service that may handle a lot of files" (such as a file storage daemon), but maybe this means something completely different?

Also, I assumed that `DynamicUser` is not applicable since this is a user service, do you know if that is correct?

Reply:
That comment no longer applies. NixOS rolls a id on first user activation and then reuses that. Currently static ids are only used if the id is hardcoded into the program and hard to change.

You can just create a user with users.users and it will get a stable id for the given machine.

Reply:
Got it, removed.